ree/reeSizeField.cc

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (C) 2011 Scientific Computation Research Center
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+#include "apfElement.h"
+#include "crv.h"
```

Comment:
@Samiullah-Malik

I don't think you need anything from from the crv here. If that's true, remove the header files that start with crv.

Reply:
Removed.